internal/testrand/rand.go

```diff
@@ -115,3 +115,95 @@ func UUID() uuid.UUID {
 	Read(uuid[:])
 	return uuid
 }
+
+// BucketName creates a random bucket name.
+//
+// Bucket names follow the restrictions set by Amazon S3:
+// https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html
+//
+// * Bucket names must be unique across all existing bucket names in Amazon S3.
```

Comment:
this rule doesn't apply to Storj bucket

Reply:
Is there a canonical design doc I can link to instead?

Reply:
I'm afraid no

Reply:
can you remove this bullet? and add information that we follow the rest of the restrictions?